src/main/java/htsjdk/samtools/liftover/LiftOver.java

```diff
@@ -103,6 +111,11 @@ public LiftOver(File chainFile) {
         }
     }
 
+    private static InputStream toInputStream(File chainFile) {
```

Comment:
If you make the change I suggested above you can remove this method, but you probably want to push the assertFileIsReadable check down into `Chain.loadChains(File)`